{"name": "Тест загрузки МК", "status": "broken", "statusDetails": {"message": "Exception: Элемент не найден", "trace": "self = <pages.sign_in_page.SignInPage object at 0x0000026EA0DFA190>\nlocator = ('css selector', '#root > div.MuiBox-root.css-18cq61h > div:nth-child(3) > div > div > div:nth-child(1) > div > div > div > div.MuiGrid-root.css-rfnosa > div > button > input[type=file]')\nkeys = 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\DatcyEstateAT_v.0.1\\\\IIFA_Economics_Auto-tests_v.0.3\\\\files\\\\МК_Upstream_v22.6Б_6+6_ЖИ_корр_штрафы_Бур_23_ЖИ.xlsx'\n\n    def send_keys_to_hidden_element(self, locator, keys):\n        try:\n>           element = Wait(self.driver, 5).until((ec.presence_of_element_located(locator)))\n\n..\\pages\\base_page.py:82: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"e1a17217cc3d85e12fcce2f2454a6a4a\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x0000026EA0DE54E0>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \t(No symbol) [0x00BC37D3]\nE       \t(No symbol) [0x00B58B81]\nE       \t(No symbol) [0x00A5B36D]\nE       \t(No symbol) [0x00A8D382]\nE       \t(No symbol) [0x00A8D4BB]\nE       \t(No symbol) [0x00AC3302]\nE       \t(No symbol) [0x00AAB464]\nE       \t(No symbol) [0x00AC1215]\nE       \t(No symbol) [0x00AAB216]\nE       \t(No symbol) [0x00A80D97]\nE       \t(No symbol) [0x00A8253D]\nE       \tGetHandleVerifier [0x00E3ABF2+2510930]\nE       \tGetHandleVerifier [0x00E68EC1+2700065]\nE       \tGetHandleVerifier [0x00E6C86C+2714828]\nE       \tGetHandleVerifier [0x00C73480+645344]\nE       \t(No symbol) [0x00B60FD2]\nE       \t(No symbol) [0x00B66C68]\nE       \t(No symbol) [0x00B66D4B]\nE       \t(No symbol) [0x00B70D6B]\nE       \tBaseThreadInitThunk [0x75A800F9+25]\nE       \tRtlGetAppContainerNamedObjectPath [0x77B77BBE+286]\nE       \tRtlGetAppContainerNamedObjectPath [0x77B77B8E+238]\n\n..\\..\\..\\..\\AppData\\Local\\pypoetry\\Cache\\virtualenvs\\iifa-economics-auto-tests-v-0-3-eUzg4zEH-py3.11\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_calculation_constructor.TestCalculationConstructor object at 0x0000026EA072CD90>\nsign_in_to_stand = <pages.sign_in_page.SignInPage object at 0x0000026EA0DFA190>\nmb_name = 'МК_Upstream_v22.6Б_6+6_ЖИ_корр_штрафы_Бур_23_ЖИ'\nclear_after_upload = True\n\n    @allure.feature(\"Загрузка МК на фронт\")\n    @allure.story(\"Загрузка всех имеющихся МК на стенд и проверка, что все загрузилось корректно\")\n    @allure.title(\"Тест загрузки МК\")\n    @allure.severity(allure.severity_level.BLOCKER)\n    @allure.description(f\"Тест запустил - {tester_name}. \\nСтенд, на котором запускался тест - {current_stand}\\nДата \"\n                        f\"запуска: {dt.now()}\")\n    @pytest.mark.calculacion_constructor\n    @pytest.mark.parametrize('mb_name', [CCFiles.MB_GEE_filename, CCFiles.MB_KUV_filename, CCFiles.MB_CNT_filename,\n                                         CCFiles.MB_YUUNG_filename, CCFiles.MB_YAG_filename,\n                                         CCFiles.MB_YUUNG_BUR_filename])\n    def test_parser(self, sign_in_to_stand, mb_name, clear_after_upload=True):\n        with allure.step(\"Прохождение авторизации на стенде\"):\n            page = sign_in_to_stand\n        with allure.step(\"Переход в конструктор расчетов\"):\n            CCPage.move_to_calculation_constructor_page(page)\n            sleep(1)\n            CCPage.is_page_calculation_page(page)\n        if clear_after_upload:\n            with allure.step(\"Очистка мастер-книг со стенда\"):\n                CCPage.delete_all_masterbooks(page)\n        with allure.step(f'Загрузка мастер-книги на стенд'):\n>           CCPage.upload_mb(page, mb_name)\n\ntest_calculation_constructor.py:51: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pages\\calculation_constructor_page.py:163: in upload_mb\n    self.send_keys_to_hidden_element(CCLocators.upload_MB, file)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.sign_in_page.SignInPage object at 0x0000026EA0DFA190>\nlocator = ('css selector', '#root > div.MuiBox-root.css-18cq61h > div:nth-child(3) > div > div > div:nth-child(1) > div > div > div > div.MuiGrid-root.css-rfnosa > div > button > input[type=file]')\nkeys = 'C:\\\\Users\\\\dpodolny\\\\PycharmProjects\\\\DatcyEstateAT_v.0.1\\\\IIFA_Economics_Auto-tests_v.0.3\\\\files\\\\МК_Upstream_v22.6Б_6+6_ЖИ_корр_штрафы_Бур_23_ЖИ.xlsx'\n\n    def send_keys_to_hidden_element(self, locator, keys):\n        try:\n            element = Wait(self.driver, 5).until((ec.presence_of_element_located(locator)))\n            return element.send_keys(keys)\n        except TimeoutException:\n>           raise Exception(\"Элемент не найден\")\nE           Exception: Элемент не найден\n\n..\\pages\\base_page.py:85: Exception"}, "description": "Тест запустил - Данил Подольный. \nСтенд, на котором запускался тест - http://genrestest.nntc.pro/economics/\nДата запуска: 2023-03-09 10:02:47.441736", "steps": [{"name": "Прохождение авторизации на стенде", "status": "passed", "start": 1678330973504, "stop": 1678330973504}, {"name": "Переход в конструктор расчетов", "status": "passed", "start": 1678330973504, "stop": 1678330974826}, {"name": "Очистка мастер-книг со стенда", "status": "passed", "start": 1678330974826, "stop": 1678330979923}, {"name": "Загрузка мастер-книги на стенд", "status": "broken", "statusDetails": {"message": "Exception: Элемент не найден\n", "trace": "  File \"C:\\Users\\dpodolny\\PycharmProjects\\DatcyEstateAT_v.0.1\\IIFA_Economics_Auto-tests_v.0.3\\tests\\test_calculation_constructor.py\", line 51, in test_parser\n    CCPage.upload_mb(page, mb_name)\n  File \"C:\\Users\\dpodolny\\PycharmProjects\\DatcyEstateAT_v.0.1\\IIFA_Economics_Auto-tests_v.0.3\\pages\\calculation_constructor_page.py\", line 163, in upload_mb\n    self.send_keys_to_hidden_element(CCLocators.upload_MB, file)\n  File \"C:\\Users\\dpodolny\\PycharmProjects\\DatcyEstateAT_v.0.1\\IIFA_Economics_Auto-tests_v.0.3\\pages\\base_page.py\", line 85, in send_keys_to_hidden_element\n    raise Exception(\"Элемент не найден\")\n"}, "start": 1678330979923, "stop": 1678330990027}], "attachments": [{"name": "stdout", "source": "94d20cfb-cebf-4e92-a821-968e90f43bde-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "mb_name", "value": "'МК_Upstream_v22.6Б_6+6_ЖИ_корр_штрафы_Бур_23_ЖИ'"}], "start": 1678330973190, "stop": 1678330990027, "uuid": "ea3d0845-9ced-4f50-9633-83bd3a40f6ea", "historyId": "59f69a094b272b3475f0ae18293cc31c", "testCaseId": "649e79659ec5cf1411c309c586996b64", "fullName": "tests.test_calculation_constructor.TestCalculationConstructor#test_parser", "labels": [{"name": "severity", "value": "blocker"}, {"name": "story", "value": "Загрузка всех имеющихся МК на стенд и проверка, что все загрузилось корректно"}, {"name": "feature", "value": "Загрузка МК на фронт"}, {"name": "tag", "value": "calculacion_constructor"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_calculation_constructor"}, {"name": "subSuite", "value": "TestCalculationConstructor"}, {"name": "host", "value": "C100119"}, {"name": "thread", "value": "5848-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_calculation_constructor"}]}